docs/index.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+entries:
+  bytebase:
+  - apiVersion: v2
+    created: "2022-11-04T10:24:46.708563+08:00"
+    description: Safe database schema change and version control for DevOps teams.
+    digest: 9b514e826fb25120f5d5fe4cb49e54323150f4f1d4ffe55fea34f9e2ed0115dc
+    home: https://bytebase.com/
+    icon: https://raw.githubusercontent.com/bytebase/bytebase.com/main/assets/logo-full.svg
+    maintainers:
+    - email: support@bytebase.com
+      name: Bytebase
+    name: bytebase
+    sources:
+    - https://github.com/bytebase/bytebase
+    type: application
+    urls:
+    - bytebase-1.0.0.tgz
```

Comment:
why we use 1.0.0 and 0.0.1 respectively here

Reply:
1.0.0 is the version of chart, the version of Bytebase should be independent from the chart version.

Reply:
The reason why I bump the version to 1.0.0 is that the 1.0.0 more like a formal version.